### DISCUSSION TITLE
Magisk 26.3

### DISCUSSION THREAD

**Discussion by CCtCa0LDRgNC00LjQ** [Genymobile/feedback#70] [Genymotion Desktop] [answered]
Good afternoon, I want to use your emulator for my project, at the moment there are 60 servers on the line, and it works on the mumu emulator, but it’s bad, yours is better.

Tell me if it is possible to integrate magisk 26.3, it is necessary for my software to work.

That is, I need customization that will be useful to your other users, and I am ready to pay in order to launch the project as soon as possible.

Is it possible to somehow solve this issue, tell me where to write and who to talk to about this topic

**Comment by mmoadvert** [comment]
I am also interested in supporting magisk 26.3 on vm with android 12 and above. Right now only old version 23 is available, it is useless for work.

Please don't delay with the update. Your VM is good, but many people do not use it precisely because of the inability to put magisk current version. And by the way, all other VMs handle it without problems (nox, memu, ldplayer, bluestacks).

**Comment by SerheoReutty** [comment]
Greetings, my team and I are considering Genimotion for work. Very interestingly, it is the only emulator that can play many smartphone models. But there is a huge disadvantage is the lack of support for Magisk 26.3+. Without it, the possibilities are very limited. We can not install the necessary modules and LSposed. In this you are far behind your competitors. Please work on this.

**Comment by Nick-2711** [comment]
I am interested in supporting magisk 25+ on vm with android 12 and above. Now only the old version 23 is available, it is useless for work. This is very necessary for my project, please do it finally, we really hope for you! I am also ready to participate financially.

**Comment by jmaneyrol69** [comment]
cc @agavignet @JulienBolard

**Comment by ph33rtehgd** [comment]
+1 for this. Magisk 23 is not nearly as useful as it once was.

**Comment by xElkomy** [comment]
there is no support for magisk 23+ until now on genymotion it so bad I think

**Comment by jmaneyrol69** [comment, marked as answer]
Hi @CCtCa0LDRgNC00LjQ @ph33rtehgd @xElkomy @mmoadvert ,
I would like to let you know that our dev team is actively working on porting Magisk 26, and maybe 28 if everything goes well.
We do not have an ETA for the moment, but we will let you know when it is ready!
Thank you for your patience.

**Comment by jmaneyrol69** [comment]
Hi @CCtCa0LDRgNC00LjQ @ph33rtehgd @xElkomy @mmoadvert , it has been a long wait but we have finally released an up-to-date build of Magisk (30.7) for Android 12+ images!

You will find the build, and the instructions to install it, in our on-line tutorial: https://www.genymotion.com/blog/tutorial/magisk-genymotion/

Do not hesitate to share your feedback with us.
